src/topology/category/Top/limits.lean

```diff
@@ -987,9 +987,8 @@ theorem nonempty_sections_of_fintype_inverse_system
   [Π (j : Jᵒᵖ), fintype (F.obj j)] [Π (j : Jᵒᵖ), nonempty (F.obj j)] :
   F.sections.nonempty :=
 begin
-  tactic.unfreeze_local_instances,
   by_cases h : nonempty J,
```

Comment:
This should be `casesI is_empty_or_nonempty J`, which will make the proof below shorter too.

Reply:
I don't understand this code enough to apply your suggestion.